explorer/src/components/Staking/MyWithdrawals.tsx

```diff
@@ -15,55 +15,77 @@ interface MyUnlockedWithdrawalsProps {
   handleAction: (action: OperatorAction) => void
 }
 
+type MyUnlockedWithdrawal = {
+  operatorId: {
+    value: string
+    sortValue: number | bigint
+  }
+  totalWithdrawalAmount: {
+    value: string
+    sortValue: number | bigint
+  }
+  totalStorageFeeRefund: {
+    value: string
+    sortValue: number | bigint
+  }
+  unlockAtConfirmedDomainBlockNumber: {
+    value: string
+    sortValue: number | bigint
+  }
+  total: {
+    value: string
+    sortValue: number | bigint
+  }
+}
+
 export const MyUnlockedWithdrawals: FC<MyUnlockedWithdrawalsProps> = ({ action, handleAction }) => {
   const { subspaceAccount } = useWallet()
   const { section, network } = useChains()
   const { withdrawals } = useConsensusStates()
-  const myUnlockedWithdrawals = useMemo(
-    () =>
-      withdrawals
-        .filter(
-          (w) =>
-            w.account === subspaceAccount &&
-            w.totalWithdrawalAmount > BIGINT_ZERO &&
-            w.withdrawals.length > 0,
+
+  const myUnlockedWithdrawals = useMemo(() => {
```

Comment:
You had mentioned that you thought the issue getting pending withdrawals that have passed the withdrawal block number to be ready to unlock was fixed in this PR but I am still seeing the same behavior as before.
![image](https://github.com/user-attachments/assets/25801606-6371-45a2-aef6-974667d8749e)

Reply:
I improved the logic around the actions listed in the actions dropdown, but I removed these 2 sections for now, still trying to think of a better way to control time, delay and such